File: python/bronze/10813.py
# 처음에는 2개 받음
N, M = map(int, input().split())
# 리스트 제작
N_list = [i+1 for i in range(N)]
for _ in range(M):
    temp1, temp2 = map(int, input().split())
    a, b = temp1-1, temp2-1
    N_list[a], N_list[b] = N_list[b], N_list[a]
for i in range(N):
    print(N_list[i], end = " ")

File: C++/bronze/10813.cpp
#include <iostream>
#include <vector>

using namespace std;

void swap_list(int &a, int &b, vector<int> &list){
    int temp = list[a];
    list[a] = list[b];
    list[b] = temp;    
}

int main(){
    int M,N, temp1, temp2;
    cin >> N >> M;
    // 제작
    vector<int> bucket(N);
    for(int i = 0; i < N; i++){
        bucket[i] = i+1;
    }
    // 반복 시작
    for(int i = 0; i < M; i++){
        cin >> temp1 >> temp2;
        temp1--;
        temp2--;
        swap_list(temp1, temp2, bucket);
    }
    for(int i = 0; i <N; i++){
        cout << bucket[i] << " ";
    }
    return 0;
}